example/format.dart

```diff
@@ -116,8 +116,15 @@ Future<void> _runTest(
   var actualText = actual.textWithSelectionMarkers;
   if (!testFile.isCompilationUnit) actualText += '\n';
 
-  // TODO(rnystrom): Handle multiple outputs.
-  var expectedText = formatTest.outputs.first.code.textWithSelectionMarkers;
+  String expectedText;
+  switch (formatTest) {
+    case UnversionedFormatTest():
+      expectedText = formatTest.output.code.textWithSelectionMarkers;
+    case VersionedFormatTest():
+      // Pick the newest style for the expectation.
+      expectedText =
+          formatTest.outputs.entries.last.value.code.textWithSelectionMarkers;
+  }
```

Comment:
Can this be a switch expression?
```suggestion
  var expectedText = switch (formatTest) {
    UnversionedFormatTest() => formatTest.output.code.textWithSelectionMarkers,
    // Pick the newest style for the expectation.
    VersionedFormatTest() => formatTest.outputs.entries.last.value.code.textWithSelectionMarkers
  };
```